client/ayon_resolve/api/constants.py

```diff
@@ -1,19 +1,19 @@
 # Ayon sequential rename variables
-rename_index = 0
-rename_add = 0
+#rename_index = 0
+#rename_add = 0
```

Comment:
Commented lines should start with a space after the `#` according to PEP style guide... (yes, very nitpicky; sorry!)
```suggestion
# rename_index = 0
# rename_add = 0
```


However, aside of that. Is there a reason these are now commented but still left in? Should they be removed?